benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/Args.kt

```diff
@@ -38,6 +40,8 @@ object Args {
      * with values separated by commas.
      */
     fun parseArgs(args: Array<String>) {
+        // reset the previous configuration before setting a new one for cases when parseArgs is called more than once:
+        reset()
```

Comment:
I would call reset explicitly from the places where it is really required.

Reply:
fixed